utils/clflags.ml

```diff
@@ -440,6 +439,43 @@ module Flambda = struct
   module Debug = struct
     let concrete_types_only_on_canonicals = ref false
   end
+
+  let oclassic_flags () =
+    cse_depth := 2;
+    join_points := true;
```

Comment:
This should say `join_points := false` for `-Oclassic`.

Reply:
done